Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -2052,7 +2057,24 @@ Document
             let renderNode = try XCTUnwrap(DocumentationNodeConverter(bundle: bundle, context: context).convert(node))
             
             // Verify task group link is beta
-            XCTAssertEqual((renderNode.references["doc://org.swift.docc.example/documentation/MyKit/MyClass"] as? TopicRenderReference)?.isBeta, true)
+            XCTAssertEqual((renderNode.references["doc://org.swift.docc.example/documentation/MyKit/globalFunction(_:considering:)"] as? TopicRenderReference)?.isBeta, true)
+        }
+        
+        // Set all platforms to beta & the exact version MyClass is being introduced.
+        // Expect the symbol to no be in beta sinceit does not have an introduced version for iOS
+        do {
+            let (bundle, context, reference) = try makeTestBundle(currentPlatforms: [
+                "macOS": PlatformVersion(VersionTriplet(10, 15, 0), beta: true),
+                "watchOS": PlatformVersion(VersionTriplet(6, 0, 0), beta: true),
+                "tvOS": PlatformVersion(VersionTriplet(13, 0, 0), beta: true),
+                "iOS": PlatformVersion(VersionTriplet(13, 0, 0), beta: true)
+            ], referencePath: "/documentation/MyKit")
+            
+            let node = try context.entity(with: reference)
+            let renderNode = try XCTUnwrap(DocumentationNodeConverter(bundle: bundle, context: context).convert(node))
+            
+            // Verify task group link is beta
```

Comment:
Comment is misleading? Maybe you copied it from above. You're testing the link is not in beta here.

Reply:
Yes! thanks for spotting this. Resolved in https://github.com/swiftlang/swift-docc/pull/1086/commits/7571b77740180af7a1ca74fad37a244de92adfff